github.js

```diff
@@ -31,19 +32,31 @@ async function constructPrMsg(sg, branch) {
  * @param {Object.<string, {title: string, pr: number}>} branchToPrDict
  * @param {string} currentBranch
  * @param {string} combinedBranch
+ * @param {'text'|'table'} format
  */
-function constructTrainNavigation(branchToPrDict, currentBranch, combinedBranch) {
-  let contents = '<pr-train-toc>\n\n#### PR chain:\n';
-  contents = Object.keys(branchToPrDict).reduce((output, branch) => {
-    const maybeHandRight = branch === currentBranch ? '👉 ' : '';
+function constructTrainNavigation(branchToPrDict, currentBranch, combinedBranch, format) {
+  let contents = '<pr-train-toc>\n\n';
+  let tableData = [['', 'PR', 'Description']];
+  Object.keys(branchToPrDict).forEach((branch) => {
```

Comment:
I actually think the `table` format reads quite a lot nicer than the `text` one - so I don't think we need to support both. Could we remove the format option and simply always render a table? That should simplify things a bit. WDYT?